README.md

```diff
@@ -129,67 +132,67 @@ public class DefaultController : Controller
 }
 ```
 
-Если нужно внутри Razor-шаблона вставить БЭМ-блок, используйте хелпер `@Html.Bem`, передав ему нужный bemjson.
+If you need to render BEM-block from the Razor template you have to use the `@Html.Bem` helper and provide the essential bemjson for it.
 ```cs
 @Html.Bem(new { block = "my-block", data = Model })
 ```
 
-В разделе `bemSettings` файла Web.config вы можете настраивать, каким способом будут выбираться бандлы с bemhtml шаблонами для http-запросов. Возможны 3 варианта мэппинга запросов на бандлы:
+In the `bemSettings` section of Web.config file you can setup how to selected the template-bundles for http-requests. There are 3 choices of request-to-bundle mapping available:
```

Comment:
-> In the `bemSettings` section of the Web.config file, you can set up how to select template bundles for HTTP requests. There are three possible ways to map requests to bundles: